chutney/ui/src/app/modules/scenarios/components/execution/history/scenario-executions-history.component.scss

```diff
@@ -9,7 +9,7 @@
 }
 
 .executions-tabs {
-    top: -1.5rem;
+    top: 1rem;
```

Comment:
Here, i would write :
`top: calc(40px - 1.5rem);`

To underline the calculation between execution header and app_main_inner elements heights.